sound/soc/sof/topology.c

```diff
@@ -2078,127 +2078,21 @@ static int sof_widget_load_siggen(struct snd_soc_component *scomp, int index,
 	return ret;
 }
 
-static int sof_get_control_data(struct snd_soc_component *scomp,
```

Comment:
@juimonen the change is legit but thecommit message is misleading. The kcontrol data will be sent at the time of widget setup which will be right after topology parsing for static pipelines and during PCM hw_params for dynamic pipelines.

Reply:
ok, commit message updated... hope it is little bit more clear